addon/services/notifications.js

```diff
@@ -11,7 +11,10 @@ const notificationAssign = assign || merge;
 const globals = config['ember-cli-notifications'] || {}; // Import app config object
 
 export default Service.extend({
-  content: A(),
+  init() {
+    this._super(...arguments);
+    this.set('content', A());
+  }
```

Comment:
```suggestion
  },
```

Reply:
🙃  😂

Reply:
lol, that's what I get for copy pasting from a ES6 class :D